evm/vm/forks/spurious_dragon/__init__.py

```diff
@@ -3,7 +3,10 @@
 from .state import SpuriousDragonState
 
 SpuriousDragonVM = HomesteadVM.configure(
+    # class name
     __name__='SpuriousDragonVM',
+    # fork name
+    fork='spurious-dragon',
```

Comment:
Do these fork names follow any pre-existing standard?

It is nice that the formatting is internally consistent, but it would be even better to make them compatible with something external out of the gate, either code or documentation. My first instinct is that we could name them the same way that ethereum/tests names the forks.

Reply:
Currently the fork names have all used their *fancy* name.  see the `evm.vm.forks` module.